trie/trieStorageManager.go

```diff
@@ -518,7 +518,8 @@ func (tsm *trieStorageManager) Remove(hash []byte) error {
 	return storer.RemoveFromCurrentEpoch(hash)
 }
 
-func (tsm *trieStorageManager) isClosed() bool {
+// IsClosed returns true if the trie storage manager signals that it has been closed
```

Comment:
[comment]
Could have simplified to :

```suggestion
// IsClosed returns true if the trie storage has been closed
```

Reply:
done